### PR TITLE
Adding restaurant tables

### PR DESCRIPTION
**Pull request by ridonkz**

### CODE REVIEW

File: liveIT-db/schema/restaurants/Cuisine.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE Cuisine (
+id SMALLINT AUTO_INCREMENT,
+type VARCHAR(50) NOT NULL,
+PRIMARY KEY(id)
+) ENGINE = INNODB;
```
Comment (riteshdalvi270) [inline liveIT-db/schema/restaurants/Cuisine.sql]: what is this used for?

File: liveIT-db/schema/restaurants/Restaurants.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
+CREATE TABLE Restaurants (
+id INT AUTO_INCREMENT,
+name VARCHAR(200) NOT NULL,
+address VARCHAR(250) NOT NULL,
+visitedDate DATETIME,
```
Comment (riteshdalvi270) [inline liveIT-db/schema/restaurants/Restaurants.sql]: default it to getDate here

File: liveIT-db/schema/restaurants/Restaurants.sql [hunk 3]
```diff
@@ -0,0 +1,19 @@
+CREATE TABLE Restaurants (
+id INT AUTO_INCREMENT,
+name VARCHAR(200) NOT NULL,
+address VARCHAR(250) NOT NULL,
```
Comment (riteshdalvi270) [inline liveIT-db/schema/restaurants/Restaurants.sql]: address would take more than 250. should we have state column separately?

File: liveIT-db/schema/restaurants/Cuisine.sql (hunk 1 above)
Comment (ridonkz) [reply liveIT-db/schema/restaurants/Cuisine.sql]: its precautionary! Storage engine used for mySQL. I can remove it...

File: liveIT-db/schema/restaurants/Restaurants.sql [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+visitedDate DATETIME,
+webURL VARCHAR(500),
+yelpURL VARCHAR(500),
+cuisineId INT NOT NULL,
+ourRating INT NOT NULL,
+expense  VARCHAR(4),
+amtPaid DECIMAL(5,2) NOT NULL,
+isClosed BOOLEAN NOT NULL,
```
Comment (riteshdalvi270) [inline liveIT-db/schema/restaurants/Restaurants.sql]: keep default as False

File: liveIT-db/schema/restaurants/Restaurants.sql (hunk 3 above)
Comment (ridonkz) [reply liveIT-db/schema/restaurants/Restaurants.sql]: we discussed we are not doing state separately.

File: liveIT-db/schema/restaurants/Restaurants.sql (hunk 4 above)
Comment (ridonkz) [reply liveIT-db/schema/restaurants/Restaurants.sql]: ok

File: liveIT-db/schema/restaurants/Restaurants.sql (hunk 2 above)
Comment (ridonkz) [reply liveIT-db/schema/restaurants/Restaurants.sql]: ok

**Pull request merged**